portfolio/src/main/webapp/index.html

```diff
@@ -227,17 +246,20 @@ <h2 class="subtitle is-italic"> My portfolio! </h2>
       </div>
     </section>
 
-    <!-- next set of boxes -->
-
+    <!-- This is the third section of the page about my achievements, important life skills, and languages I speak. This is again divided into two columns, the left column has achievements in an article with boxes
+    and the second columns has two articles (one for skills and the other for languages). Inside each article, there are boxes, and each box has an article within it to talk about each instance/event. -->
```

Comment:
For multiple line comments, start the sentence with a new line and indent by 2 spaces.

\<!--
  This is the third section of the page about my achievements, important life skills, and languages I speak. This is
  again divided into two columns, the left column has achievements in an article with boxes and the second columns
  has two articles (one for skills and the other for languages). Inside each article, there are boxes, and each box has
  an article within it to talk about each instance/event.
-->